requirements_dev.txt

```diff
@@ -9,3 +9,4 @@ Sphinx==1.4.4
 cryptography==1.4
 PyYAML==3.11
 pytest==2.9.2
+py-trello==0.5.0
```

Comment:
I might be wrong, but this should be in setup.py.
requirements_dev.txt I think is for packages that help us to develop and are not needed for shipping trilla itself.